cogboard-webapp/src/components/widgets/types/ServiceCheckWidget.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {Typography} from "@material-ui/core";
+
+
+const ServiceCheckWidget = props => {
+  const {statusCode, statusMessage, timestamp } = props;
+  const ts = timestamp ? new Date(timestamp).toLocaleString() : '';
+
+  return (
+    <>
+      <Typography
+        variant="caption">
+        <p>{ts}</p>
+        <p>{statusMessage}</p>
+      </Typography>
+      <Typography
+        variant="h4">
+        {statusCode}
+      </Typography>
+    </>
+  );
+};
+
```

Comment:
Add propTypes and defaultProps if needed. https://pl.reactjs.org/docs/typechecking-with-proptypes.html